### ISSUE TITLE
bench: broadcast page shouldn't resubmit form on reload.

### ISSUE THREAD

**Issue by trekhopton** [ausocean/cloud#104]
When a form is submitted, it's generally good practice to either go to another URL, or clear the form and display a success message. 
We should implement one of these options on the broadcast page.

I propose we initially have the broadcast config displayed as a view only UI with labels, then when a user clicks "edit" it loads admin/broadcast/edit and turns the config labels into config form inputs and they can be edited. A user can then either click "save" which submits the new config and loads admin/broadcast, or cancel, which takes them back to admin/broadcast without saving.

**Comment by scruzin** [comment]
Is this fixed @trekhopton

**Comment by trekhopton** [comment]
No, it's not although it's more of a desirable change than a bug, not very high priority.

**Comment by deborah-baker** [comment]
This caused an issue where multiple streams were started because the browser refresh was used @ao-david
